src/rpc/mining.cpp

```diff
@@ -663,7 +663,7 @@ static RPCHelpMan getblocktemplate()
     if(!node.connman)
         throw JSONRPCError(RPC_CLIENT_P2P_DISABLED, "Error: Peer-to-peer functionality missing or disabled");
 
-    if (node.connman->GetNodeCount(CConnman::CONNECTIONS_ALL) == 0)
+    if (node.connman->PeerCount() == 0)
```

Comment:
style: add braces to comply with current style when touching old code.

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/86#issuecomment-691216729).